addons/point_of_sale/models/report_sale_details.py

```diff
@@ -189,7 +189,7 @@ def get_sale_details(self, date_start=False, date_stop=False, config_ids=False,
                         payment['cash_moves'] = cash_in_out_list
                         payment['count'] = True
             if not is_cash_method:
-                cash_name = 'Cash ' + str(session.name)
+                cash_name = _('Cash ') + str(session.name)
```

Comment:
```suggestion
                cash_name = _('Cash') + " " + str(session.name)
```
So we do'nt need to add a new translation.